apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -198,16 +205,22 @@ handle_reconcile(VIPs, #state{tree=Tree, nodes=Nodes, namespaces=Namespaces,
 
             PrevRoutes = get_routes(RouteMgr, Namespace),
             DiffRoutes = dcos_net_utils:complement(Routes, PrevRoutes),
+            prometheus_counter:inc(
+                l4lb, reestablished_routes_total, [], dcos_net_utils:complement_size(DiffRoutes)),
```

Comment:
I am not sure this is the right approach here, I just assumed when there are no changes to be made then there will be no complement as A == B. If that is not what you had in mind, could you point me to how would you do it?